src/Microsoft.Health.Fhir.Shared.Api/Features/Formatters/HttpContextExtensions.cs

```diff
@@ -48,7 +48,7 @@ public static bool GetIsPretty(this HttpContext context)
             {
                 if (!bool.TryParse(query, out bool isPretty))
                 {
-                    // Assume no pretty formatting if parameter can't be parsed.
+                    // ContentTypeService validates the _pretty parameter. This is reached if other errors are encountered when parsing the query string.
```

Comment:
What other errors actually cause this?

Reply:
If both `format=blah` and `pretty=blah`, this will be reached when the operation outcome for the error is being formatted.